### PR TITLE
Clean up query descriptions

### PR DESCRIPTION
**Pull request by Noratrieb**
Use the same tense everywhere and prefer display over debug, as these descriptions are user facing.

### CODE REVIEW

File: compiler/rustc_middle/src/query/mod.rs [hunk 1]
```diff
@@ -719,20 +719,20 @@ rustc_queries! {
     ///                         // DefId
     /// trait Trait {           // trait_id
     ///     fn f();             // trait_f
-    ///     fn g() {}           // trait_g
+    ///     fn g() `{}`           // trait_g
```
Comment (oli-obk) [inline compiler/rustc_middle/src/query/mod.rs:722]: Overzealous replacement. There are more of these below

Comment (Nilstrieb) [reply compiler/rustc_middle/src/query/mod.rs:722]: Oops, find and replace was a little to aggressive

**Comment by oli-obk** [comment]
You'll also need to bless some ui tests

File: src/test/ui/async-await/no-const-async.stderr [hunk 2]
```diff
@@ -18,7 +18,7 @@ note: ...which requires borrow-checking `x`...
    |
 LL | pub const async fn x() {}
    | ^^^^^^^^^^^^^^^^^^^^^^
-note: ...which requires processing `x`...
+note: ...which requires processing MIR for `x`...
```
[the remark this answers is not in the document]
Comment (oli-obk) [reply src/test/ui/async-await/no-const-async.stderr:21]: We could call it either `preparing x for borrow checking` or `performing implicit promotion to static in x`

The latter is correcter, but a bit confusing

**Pull request merged**